python/ray/rllib/agents/agent.py

```diff
@@ -141,9 +141,8 @@
     # remote processes instead of in the same worker. This adds overheads, but
     # can make sense if your envs are very CPU intensive (e.g., for StarCraft).
     "remote_worker_envs": False,
-    # Similar to remote_worker_envs, but runs the envs asynchronously in the
-    # background for greater efficiency. Conflicts with remote_worker_envs.
-    "async_remote_worker_envs": False,
+    # Timeout that remote workers are waiting when polling environments
+    "remote_worker_env_timeout_ms": 0,
```

Comment:
What are good settings you've found? It might be helpful to mention how to configure this in the comment here.

Reply:
I think you should also update rllib-env.rst to remove the async config reference.

Reply:
Added small best usage description. In our case, out model used ~4.5ms on inference and our environment used ~1.5ms on step and ~700ms on reset. We put 2ms wait so we could batch inference but not wait for slow reset.